pkg/stucts/accumulated_info_template_input.go

```diff
@@ -37,9 +46,25 @@ func InitAccumulatedInfoTemplateInput(queryInfos SimilarQueryInfoList, filePaths
 	var timeStamps []*time.Time
 	totalQueryCount := 0
 	t := time.Time{}
+	qtc := QueryTypeCount{0, 0, 0, 0}
 
 	for _, info := range queryInfos {
 		totalQueryCount += info.Count
+		if info.Query != "" {
+			queryType := strings.Split(info.Query, " ")[0]
+			if queryType == "select" || queryType == "SELECT" {
```

Comment:
what if the `queryType` is `Select`?
A good way of comparing this is to "fix" the case of `queryType` value by making it either fully uppercase or lowercase and only after that comparing to the expected value.

Reply:
Updated